openfga/v1/openfga_service.proto

```diff
@@ -31,8 +31,6 @@ service OpenFGAService {
         "1. `tuple_key.object` is mandatory. It can be a full object (e.g., "
         "`type:object_id`) or type only (e.g., `type:`).\n"
         "2. `tuple_key.user` is mandatory in the case the `tuple_key.object` is a type only.\n"
-        "3. `authorization_model_id` is optional. If specified, it will be used to assert that the input `tuple_key` is valid for the model specified. "
-        "If not specified, the latest authorization model ID will be used. \n"
```

Comment:
Oh my goodness. Sorry, but now that you are here, can you also add that the tuple_key is optional and if not supplied will return all tuples in the store.

Thank you!

Reply:
Done